### DISCUSSION TITLE
Overflow error with a button

### DISCUSSION THREAD

**Discussion by enrico002** [wiremod/wire#3140] [Q&A]
Hi,
I wanted to move an object from one point to another with a button but it gives me the error:
" 'You can only perform one type of action per tick!' "
Why does the button take the same command multiple times if I press it only once?

**Comment by Denneisk** [comment]
With the old E2 trigger system, every change in input causes the E2 to run. Certain actions, mostly propcore, are limited to running only once per tick. The switch of a button from 0 to 1, and then back from 1 to 0, is a case where such a change could happen. There are also other cases, like a timer or something else in your code or setup causing the E2 to execute multiple times.

Consider guarding your actions to run only when necessary, and also consider using E2 events if you're not already, which allow a much safer way to control when code is ran.
